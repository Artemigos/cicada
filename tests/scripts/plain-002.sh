if echo hi > /dev/null
    findext 2>/dev/null || echo findext not found
fi

if echo hi > /dev/null
    ifif7 2>/dev/null || echo ifif7 not found
fi

for x in foo
    ford 2>/dev/null || echo ford not found
done

for x in foo
    doned 2>/dev/null || echo doned not found
done

counter=19
while echo "$counter" | grep -iq "^1.$"
    whiled 2>/dev/null || echo whiled not found
    counter=$(expr $counter + 1)
done
