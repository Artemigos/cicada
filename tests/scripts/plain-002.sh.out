findext not found
ifif7 not found
ford not found
doned not found
whiled not found
